ansible/roles/postgresql/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 pg_install: True
-postgresql_version: '9.1'
+postgresql_version: '9.4'
```

Comment:
Yes! Thank you! I was sure there were two places, but I only found this one. I'll update.